inc/TRestRawSignalChannelActivityProcess.h

```diff
@@ -45,34 +45,36 @@ class TRestRawSignalChannelActivityProcess : public TRestEventProcess {
     TRestDetectorReadout* fReadout;  //!
 #endif
 
+    void InitFromConfigFile() override;
+
     void Initialize() override;
 
     void LoadDefaultConfig();
 
    protected:
     /// The value of the lower signal threshold to add it to the histogram
-    Double_t fLowThreshold = 25;
+    Double_t fLowThreshold;
 
     /// The value of the higher signal threshold to add it to the histogram
-    Double_t fHighThreshold = 50;
+    Double_t fHighThreshold;
 
     /// The number of bins at the daq channels histogram
-    Int_t fDaqChannels = 300;
+    Int_t fDaqHistogramChannels;
```

Comment:
Why to remove the default values from header? Providing the default value on the headers is now the stardard way.

Reply:
I removed them because I had defined them in the .cxx file, to keep coherence with `TRestDetectorSignalChannelActivityProcess`.  But I am going to put them back here if it is the standard way now. Should I then also change the cxx, so that the default values are not defined in two places?

Reply:
You should completely remove `InitFromConfigFile` implementation.